extensions/ql-vscode/src/remote-queries/view/ExportButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Button = styled.a`
+  color: var(--vscode-button-foreground);
+  background-color: var(--vscode-button-background);
+  &:hover {
+    color: var(--vscode-button-foreground);
+    text-decoration: none;
+    background-color: var(--vscode-button-hoverBackground);
+  }
+  cursor: pointer;
+  padding: 5px 10px;
+`;
+
+const ExportButton = ({ text, onClick }: { text: string, onClick: () => void }) => (
```

Comment:
Minor: this isn't really an _export_ button. What you've made here is something more general. Maybe rename to something like `ActionButton` or `StyledButton` or ...

Reply:
Good point! The button's only used for exporting at the moment, so I'll keep the name for now (for the sake of easily remembering where it's used 🔍 ). But we could rename it in future and use it more generally! 💭

Reply:
Just my 2p.. there is value in renaming now. Think of a new user coming to our codebase. They're making some UI changes that include adding a button that executes an action. They look the current components and they see "ExportButton". At that point they might think that it's not what they want and create their own component, as what they want has nothing to do with exporting. `ActionButton` is more likely to be considered in such case.

Reply:
Makes sense, I will be less lazy and do it now! 😂